src/app/error.js

```diff
@@ -0,0 +1,16 @@
+'use client';
+
+import PropTypes from 'prop-types';
+
+export default function Error({ error }) {
+  return (
+    <div className="w-[800]">
+      <h2>Something went wrong!</h2>
+      <pre>{JSON.stringify(error, null, 4)}</pre>
```

Comment:
@alex-popov-tech what do you think about Sentry integration?

Reply:
Will need to discuss on how to handle errors on the user side with the team.

Reply:
@alex-popov-tech

Reply:
hello, sorry for missing this out

sure, we can use sentry, or any other tool to log/store/trace errors in both be/fe, altho this is a bit out of scope, we need to create separate task for this

page for errors is still needed i think, but its also out of scope of current PR